src/_data/sidenav.yml

```diff
@@ -274,8 +281,6 @@
           permalink: /development/tools/flutter-fix
         - title: Code formatting
           permalink: /development/tools/formatting
-        - title: Web renderers
-          permalink: /development/tools/web-renderers
```

Comment:
Should we have redirects added for the old page?

Reply:
Yes, I was going to ask John for that. :D